web/src/components/molecules/Content/RequestCreationModal/index.tsx

```diff
@@ -5,39 +5,64 @@ import Input from "@reearth-cms/components/atoms/Input";
 import Modal from "@reearth-cms/components/atoms/Modal";
 import Select from "@reearth-cms/components/atoms/Select";
 import TextArea from "@reearth-cms/components/atoms/TextArea";
+import { Member } from "@reearth-cms/components/molecules/Workspace/types";
+import { RequestState } from "@reearth-cms/gql/graphql-client-api";
```

Comment:
***IMPORTANT: No graphql imports in molecules or atoms.
Recreate the type locally and/or convert the variable to a typed javascript object/variable before passing it to the molecule